diff --git a/src/materials/Material.d.ts b/src/materials/Material.d.ts
index 731bcc89e166bc9cfce9746ba4c599b47faa24cc..b673aa9bb2b41e484895dd53e849b253da5084bb 100644
--- a/src/materials/Material.d.ts
+++ b/src/materials/Material.d.ts
@@ -16,12 +16,13 @@ import { BufferGeometry } from "../core/BufferGeometry.js";
 import { EventDispatcher } from "../core/EventDispatcher.js";
 import { JSONMeta, Object3D } from "../core/Object3D.js";
 import { Color, ColorRepresentation } from "../math/Color.js";
+import { Vector3 } from "../math/Vector3.js";
 import { Plane } from "../math/Plane.js";
 import { Group } from "../objects/Group.js";
 import { WebGLProgramParametersWithUniforms } from "../renderers/webgl/WebGLPrograms.js";
 import { WebGLRenderer } from "../renderers/WebGLRenderer.js";
 import { Scene } from "../scenes/Scene.js";
-import { EulerTuple, SourceJSON, TextureJSON, Vector2Tuple } from "../Three.js";
+import { EulerTuple, SourceJSON, TextureJSON, Vector2Tuple, Vector4 } from "../Three.js";
 
 export interface MaterialParameters {
     alphaHash?: boolean | undefined;
@@ -53,6 +54,7 @@ export interface MaterialParameters {
     premultipliedAlpha?: boolean | undefined;
     forceSinglePass?: boolean | undefined;
     dithering?: boolean | undefined;
+    ditheredHole?: Vector4 | undefined;
     side?: Side | undefined;
     shadowSide?: Side | undefined;
     toneMapped?: boolean | undefined;
